### PR TITLE
EE-1143: Use real gas price

### PR DESCRIPTION
**Pull request by mpapierski**
Ref: https://casperlabs.atlassian.net/browse/EE-1143

### CODE REVIEW

**Comment by AlexanderLimonov** [comment]
Please see economics channel - without refunds, EE might not need to use gas price for anything, since the deploy header supplies the total charge. The tricky part is the payment code.

File: node/src/components/block_proposer/tests.rs [hunk 1]
```diff
@@ -17,14 +17,18 @@ fn default_gas_payment() -> Gas {
     Gas::from(1u32)
 }
 
+fn default_gas_price() -> u64 {
```
Comment (Fraser999) [inline node/src/components/block_proposer/tests.rs:20]: Wouldn't it be normal to have this as a const?

**Review by Fraser999 — COMMENTED** [review]
Now that we're actually using the gas price from the deploy, it might be worthwhile updating [the default value for it in the client](https://github.com/CasperLabs/casper-node/blob/master/client/src/deploy/creation_common.rs#L242) too?

**Review by dwerner — APPROVED** [review]

**Comment by mpapierski** [comment]
> 
> 
> Now that we're actually using the gas price from the deploy, it might be worthwhile updating [the default value for it in the client](https://github.com/CasperLabs/casper-node/blob/master/client/src/deploy/creation_common.rs#L242) too?

Changed from 10 to 1 so people won't suddenly pay 10x times the cost since we're now using real gas price.

**Pull request merged**